packages/components/addon/components/hds/app-footer/index.hbs

```diff
@@ -0,0 +1,10 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class="hds-app-footer" ...attributes>
+  <ul class="hds-app-footer__list">
```

Comment:
now that everything is inside a list, do we foresee any accessibility issue if the consumers put some unexpected content in a custom list item (eg. the theme selector, or other things)?

Reply:
Another (possible) downsides of this approach is that now all the items have the same spacing between them, while (I think) the list of links should be a bit tighter than the rest of the items, to differentiate the information architecture.

Maybe we could have a structure in which we have:
- `footer` - main container
- `list(s)` - one or more containers, each one with a larger space between them
- `item(s)` - one or more items, each one with a smaller space between them, to show that they are semantically grouped (eg all the links, all the metadata, etc)
In that case, we may lose control of how the items wrap, because we would have intermediate "blocks" (the "list(s)").

Just an idea, at the end this is heavily influenced by the design decisions.

Reply:
I don't agree with changing the spacing of the legal links to be tighter than the other list items unless we also want to change the design in other ways to more greatly differentiate them. Otherwise it will just look weird I think. (Although I could be wrong but this would require visual design changes and I thought we were trying to finalize and lock down the design at this point.)

I can get rid of the list markup.

Reply:
I removed/replaced the list (`<ul>`, `<li>`) mark up.

Reply:
[issue, a11y] 

1. These items _should_ be in a list, not only so the user receives a count but also so the items are grouped appropriately together and the user can navigate directly to the list. 
2. The list itself (the `<ul>` element) should have an accessible name (`aria-label` will work) so that it is able to be differentiated amongst other lists on the page.

Reply:
@didoo Would it be ok if I change it back to what I had?

I'm still confused as to how to structure and the support the various custom content options you think we should so I don't know if what I had goes against what you were thinking.

Reply:
I added back a ul and added yields for adding other custom content. I'm not sure if I'm structuring everything as you intended though. @didoo